pkg/fission-cli/package.go

```diff
@@ -209,7 +211,8 @@ func updatePackage(client *client.Client, pkg *fv1.Package, envName, envNamespac
 	if needToBuild || forceRebuild {
 		// change into pending state to trigger package build
 		pkg.Status = fv1.PackageStatus{
-			BuildStatus: fv1.BuildStatusPending,
+			BuildStatus:         fv1.BuildStatusPending,
+			LastUpdateTimestamp: time.Now().UTC(),
```

Comment:
You also need to update this function https://github.com/fission/fission/blob/b47b3ecb702a1eb7639cca2dc447fec2edc072ca/pkg/buildermgr/common.go#L126-L129

Reply:
added the `LastUpdateTimestamp`